src/Mod/Mesh/App/Exporter.cpp

```diff
@@ -54,7 +56,32 @@ std::string Exporter::xmlEscape(const std::string &input)
     return out;
 }
 
-MergeExporter::MergeExporter(std::string fileName, MeshIO::Format fmt)
+bool Exporter::addAppGroup(App::DocumentObject *obj, float tol)
+{
+    const auto meshFeatId( Base::Type::fromName("Mesh::Feature") );
+    const auto partFeatId( Base::Type::fromName("Part::Feature") );
+    const auto appPartId( Base::Type::fromName("App::Part") );
+    const auto appDOGId( Base::Type::fromName("App::DocumentObjectGroup") );
+
+    auto ret(true);
+
+    for (auto it : static_cast<App::Part *>(obj)->getOutList()) {
+        if (it->getTypeId().isDerivedFrom(meshFeatId)) {
+            ret &= addMeshFeat(it);
+        } else if (it->getTypeId().isDerivedFrom(partFeatId)) {
+            ret &= addPartFeat(it, tol);
+        } else if ( it->getTypeId().isDerivedFrom(appPartId) ||
```

Comment:
Thanks Stefan!  I'll try to get this sorted after work.  The intent is to get things that can be exported as meshes, and filter them to the appropriate handler.